package.json

```diff
@@ -12,10 +12,13 @@
     "postinstall": "cd packages/react-instantsearch && npm install",
     "doctoc": "doctoc --maxlevel 1 README.md CONTRIBUTING.md",
     "test": "(cd packages/react-instantsearch && npm test) && npm run lint && NODE_ENV=production DOCS_MOUNT_POINT=/instantsearch.js/react/ DOCS_DIST=docs/react/ npm run docs:build && NODE_ENV=production npm run build",
-    "dev": "cd packages/react-instantsearch && npm run dev"
+    "dev": "cd packages/react-instantsearch && npm run dev",
+    "storybook": "start-storybook -p 6006",
```

Comment:
Imho storybook seems to be a nice addition and will help us or any contributor. I'm wondering if it could work with jest snapshot to check visually the regressions / changes.